conf/locale/babel_mako.cfg

```diff
@@ -1,13 +1,30 @@
-# Extraction from Mako templates
-[mako: cms/templates/**.html]
+# Extraction from Mako templates.
+# Mako files can appear in a number of different places. Some we want to
+# extract strings from, some we don't.
+#
+# Extract from these directory trees:
+#
+#   cms/templates
+#   cms/djangoapps/APPNAME/templates
+#   common/templates
+#   common/djangoapps/APPNAME/templates
+#   lms/templates
+#   lms/djangoapps/APPNAME/templates
+#   common/lib/capa/capa/templates
+#
+# Don't extract from these directory trees:
+#   common/test/test_microsites/test_microsite/templates
+#
+
+[mako: */templates/**.html]
 input_encoding = utf-8
-[mako: lms/templates/**.html]
+[mako: */djangoapps/*/templates/**.html]
```

Comment:
I think it does match, I saw one string ("Teams") extracted. We can add the extra star since everyone will have that extra directory anyway, but I don't think it is needed.